tests/nimble_autoconn_ccnl/Makefile

```diff
@@ -0,0 +1,28 @@
+BOARD ?= nrf52dk
+include ../Makefile.tests_common
+
+# include shell support
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += ps
+
+# Include GNRC and RPL
+USEMODULE += gnrc_netdev_default
+USEMODULE += auto_init_gnrc_netif
+
+# Setup CCN-lite
+USEPKG += ccn-lite
+CFLAGS += -DUSE_LINKLAYER
+CFLAGS += -DUSE_RONR
+CFLAGS += -DCCNL_UAPI_H_
+CFLAGS += -DUSE_SUITE_NDNTLV
+CFLAGS += -DNEEDS_PREFIX_MATCHING
+CFLAGS += -DNEEDS_PACKET_CRAFTING
+
+# Setup Nimble
+USEMODULE += nimble_autoconn_ndnsp
+NIMBLE_MAX_CONN := 3
```

Comment:
Same as below, does it need to be `:=`.

Also can you provide a comment saying why 3?

Reply:
Its a good tradeof between memory consumption and network performance for smaller networks. There is no right and wrong here, a suitable values depends much on the use-case.

I removed these values from the Makefiles for now so the examples simple use the default value defined in the NimBLE package (which is 3 :-) )